feat(redis): add DB menu MTA-2004 (#696)
scaleway/docs-content: faq/database.mdx

---
meta:

>   title: Database

Reviewer: ```suggestion
  title: Databases
```

>   description: Discover Scaleway Database and how to access it for PostgreSQL and MySQL.

Reviewer: ```suggestion
  description: Discover Scaleway Databases and how to access them for PostgreSQL and MySQL.
```

content:

>   h1: Database

Reviewer: ```suggestion
  h1: Databases
```

>   paragraph: Discover Scaleway Database and how to access it for PostgreSQL and MySQL.

Reviewer: ```suggestion
  paragraph: Discover Scaleway Databases and how to access them for PostgreSQL and MySQL.
```

IPv6 is not supported with Private Networks on Database Instances.

> ## In what regions is the Private Networks feature available for Databases

Reviewer: ```suggestion
## In what regions is the Private Networks feature available for PostgreSQL and MySQL Databases
```

> Private Networks is available for Database Instances in PAR1, AMS1 and WAW1.

Reviewer: ```suggestion
Private Networks is available for PostgreSQL and MySQL Database Instances in PAR1, AMS1 and WAW1.
```
